R/Design-methods.R

```diff
@@ -417,6 +417,19 @@ setMethod("simulate",
           data = thisData
         )
 
+        # Get the MTD estimate from the samples.
+        sample_mtd <- dose(
+          mean(object@nextBest@target),
+          model = object@model,
+          samples = thisSamples
+        )
```

Comment:
Why is the MTD determined like this? I am wondering because we use `thisDose` elsewhere here as the MTD (see below)

Reply:
Thanks for the suggestion. If I am not mistaken it is the estimated MTD per sample, i.e. the calculated dose that corresponds to to the target (or the mean of the interval if target is an interval) for a specific sample. But I see the point that the dose is the exact dose and not likely to be a dose from the dosegrid. Therefore I think the proposal makes sense to rename it to `target_dose_samples`.